frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -32,17 +32,15 @@ action.GitMachete.GitCommandUpdatingCurrentBranchBackgroundable.notification.tit
 
 action.GitMachete.BaseSlideInBackgroundable.notification.message.timeout=Branch {0} has not been created within the allowed time.
 action.GitMachete.BaseSlideInBackgroundable.notification.message.wait-interrupted=Waiting for creation of branch {0} has been interrupted.
-action.GitMachete.BaseSlideInBackgroundable.task-title=Sliding in
-
-action.GitMachete.RenameBackgroundable.task-title=Rename branch
+action.GitMachete.BaseSlideInBackgroundable.task-title=Sliding in\u2026
 
 
 action.GitMachete.SlideInNonRootBackgroundable.notification.message.entry-does-not-exist.HTML=<html>Branch <b>{0}</b> does not exist</html>
 action.GitMachete.SlideInNonRootBackgroundable.notification.message.entry-is-descendant-of.HTML=<html>Branch <b>{0}</b> cannot be slid under branch <b>{1}</b> since <b>{0}</b> is already an ancestor of <b>{1}</b></html>
 action.GitMachete.SlideInNonRootBackgroundable.notification.title.slide-in-fail.HTML=<html>Slide in of branch <b>{0}</b> failed</html>
 
 # Quasi-title capitalization intended since we always write "Git Machete" with initial caps.
-action.GitMachete.GitMacheteRepositoryUpdateBackgroundable.task-title=Updating Git Machete status
+action.GitMachete.GitMacheteRepositoryUpdateBackgroundable.task-title=Updating Git Machete status\u2026
```

Comment:
Is it consistent with the general convention of background task names in IntelliJ? TBH I don't remember